WalletWasabi.Gui/Global.cs

```diff
@@ -72,6 +70,8 @@ public class Global
 
 		public JsonRpcServer? RpcServer { get; private set; }
 
+		private ServiceProvider Container { get; }
```

Comment:
I would like to see how this can be used in a ViewModel in a clean way without making this static or if it is possible?

Reply:
This is a good question for @danwalmsley.

Reply:
I have a few ideas but it really depends on how Avalonia is architected so I can come up with potential solutions if/after Dan can share here his point of view.